izpack-maven-plugin/src/main/java/org/izpack/mojo/IzPackNewMojo.java

```diff
@@ -253,6 +278,12 @@ private void initMavenProperties(PropertyManager propertyManager)
                 }
                 if (!containsExcludedProperty(propertyName, exclusionList))
                 {
+                    if (null != includeFilter) {
+                        if (!includeFilter.contains(propertyName)) {
```

Comment:
Since the `propertyName` come only from POM (the loop at the line no. 267 -- code snippet given below), the `propertyName` provided at the command line will not be added even though it is not the exclusion list.

```
Properties properties = project.getProperties();
...
...
for (String propertyName : properties.stringPropertyNames())
```